java/test/org/certificatetransparency/ctlog/TestData.java

```diff
@@ -15,14 +15,17 @@
   private static final String DATA_ROOT = "test/testdata/";
   // Public log key
   public static final String TEST_LOG_KEY = DATA_ROOT + "ct-server-key-public.pem";
+  public static final String TEST_LOG_KEY_RSA = DATA_ROOT + "ct-server-key-public-rsa.pem";
```

Comment:
Could you put all the new RSA-related test data under DATA_ROOT/rsa ?
Otherwise it might confuse other developers looking at the test/testdata directory to see two keys (because everything under test/testdata was generated/signed by the same log key).